From 2cc481e570a27fd7320cbdeccf281924f94cd559 Mon Sep 17 00:00:00 2001
From: Albert Chen <R65187@freescale.com>
Date: Tue, 19 Aug 2008 10:16:12 +0800
Subject: [PATCH] ENGR00088315: USB module can't pass compile in mx35.

Mx35: Because mx35 didn't define USB_IRAM_BASE_ADDR,
so failed to compile.

Signed-off-by: Albert Chen <r65187@freescale.com>
---
 drivers/usb/gadget/arcotg_udc.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/gadget/arcotg_udc.h b/drivers/usb/gadget/arcotg_udc.h
index 689b907..9048b5f 100644
--- a/drivers/usb/gadget/arcotg_udc.h
+++ b/drivers/usb/gadget/arcotg_udc.h
@@ -36,6 +36,10 @@
 #define IRAM_PPH_NTD	0
 #endif
 
+#ifndef USB_IRAM_BASE_ADDR
+#define USB_IRAM_BASE_ADDR 0
+#endif
+
 #define NEED_IRAM(ep) ((g_iram_size) && \
 	((ep)->desc->bmAttributes == USB_ENDPOINT_XFER_BULK))
 
-- 
1.5.4.4

